### PR TITLE
Bug 437431 fixing rotation prompt

### PR DESCRIPTION
**Pull request by SzymonSmykala**

### CODE REVIEW

File: iOS/FVM/FVMTests/DamagedParts/Helpers/HightlightHandlerMock.swift [hunk 1]
```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 import SceneKit
-@testable import FVM
```
Comment (lapwingg) [inline iOS/FVM/FVMTests/DamagedParts/Helpers/HightlightHandlerMock.swift]: Did you notice that Travis build failed?
Also I tried to testing app in simulator and ... also build failed
You have to restore this line :)

File: iOS/FVM/Framework/Controllers/FVMDamagedCarViewController.swift [hunk 2]
```diff
@@ -81,25 +80,21 @@ public class FVMDamagedCarViewController: UIViewController {
...
+    private func enableAcceptButton() {
         acceptButton.setTitleColor(UIColor.blue, for: .normal)
         acceptButton.alpha = 1.0
         acceptButton.isEnabled = true
     }
     
     @objc
-    fileprivate func hideRotationPrompt() {
-        for view in self.view.subviews {
-            if view.restorationIdentifier?.isEqual(ROTATION_PROMPT) ?? false as Bool {
```
Comment (lapwingg) [inline iOS/FVM/Framework/Controllers/FVMDamagedCarViewController.swift]: Const field ROTATION_PROMPT is no longer used.
Please remove its declaration from code (FVMDamagedCarViewController -> line 26)

File: iOS/FVM/FVMTests/DamagedParts/Helpers/HightlightHandlerMock.swift (hunk 1 above)
Comment (SzymonSmykala) [reply iOS/FVM/FVMTests/DamagedParts/Helpers/HightlightHandlerMock.swift]: This is strange, Xcode tells me that we don't need it

Comment (ghawliczek) [reply iOS/FVM/FVMTests/DamagedParts/Helpers/HightlightHandlerMock.swift]: Had exactly the same issue but didn't dig into it, might be something with project definition, we can check pbxproj in some spare time

**Pull request merged**